.changeset/twelve-days-hunt.md

```diff
@@ -0,0 +1,8 @@
+---
+"@kaizen/tailwind": patch
+---
+
+Update filter method for tailwind spacing tokens to allow default tailwind width values
+ - This fixes the !Nan filtered our tailwind percentile and string values
+ - Updated stories that use w-100 to w-full since the original interntion was to be compile to width: 100%
```

Comment:
```suggestion
 - Updated stories that use w-100 to w-full since the original intention was to be compile to width: 100%
```

Reply:
lul, that was not my interntion 😆 Will fix 👍